plugins/processors/dcos_metadata/dcos_metadata.go

```diff
@@ -296,16 +311,27 @@ func mapExecutorNames(ge *agent.Response_GetExecutors) map[string]string {
 	return results
 }
 
+func containsWhitelistedPrefix(key string, whitelist []string) (prefix string, contains bool) {
+	for _, pre := range whitelist {
+		if strings.HasPrefix(key, pre) {
+			return pre, true
```

Comment:
nit: If the key is equal to the prefix, this is also going succeed and the label will be empty.